README.md

```diff
@@ -12,9 +12,10 @@ You can find a number of Dashboard Plugins in this repository. Each folder conta
 
 You can find plugins that may no longer be actively used due to their functionality being incorporated into the GoodData product in [legacy plugins](./legacy_plugins/) folder. They can still serve as valuable starting points for creating similar plugins.
```

Comment:
@dremileta This is actually not true for this plugin, so could we add here that it can be for other reasons..., I mean something like `...being incorporated into the GoodData product (or other reasons) in ..

Reply:
@dremileta as mentioned here: https://github.com/gooddata/gooddata-dashboard-plugins/pull/47#pullrequestreview-1744628986, we still need to make a similar change in the legacy plugins readme (https://github.com/dremileta/gooddata-dashboard-plugins/tree/insight_to_action/legacy_plugins)